src/compiler/scala/tools/nsc/ast/parser/xml/Utility.scala

```diff
@@ -77,7 +77,7 @@ object Utility {
       else sb append c
     }
 
-    if(!sb.isEmpty) // flush buffer
+    if(sb.nonEmpty) // flush buffer
```

Comment:
oh man, missed opportunity to normalize spacing before the LPAREN. I promise to contribute more reviews on lunch breaks. Or maybe on snack breaks. Also, in a demoralizing blow, retronym recently came out against `nonEmpty`, in some context. Or is this scala StringBuilder with native nonEmpty, and why do I have to ask?

Reply:
Good question -- I didn't think to check, but we get `TraversableOnce's` version here: `def nonEmpty: Boolean = !isEmpty` (according to IntelliJ). So, neither `isEmpty` nor `nonEmpty` uses the potentially more efficient `length` on the underlying Java `StringBuilder`.